src/main/java/com/playmonumenta/networkchat/RemotePlayerListener.java

```diff
@@ -33,7 +33,9 @@ public class RemotePlayerListener implements Listener {
 	private static @Nullable RemotePlayerListener INSTANCE = null;
 	private static final Map<UUID, Component> mPlayerComponents = new HashMap<>();
 
-	private RemotePlayerListener() {}
+	private RemotePlayerListener() {
+
```

Comment:
If this change needs to be here, I'd suggest removing the empty line in between:
```java
private RemotePlayerListener() {
}
```